packages/app-store/ee/routing-forms/components/FormActions.tsx

```diff
@@ -447,7 +447,10 @@ export const FormAction = forwardRef(function FormAction<T extends typeof Button
   }
   return (
     <DropdownMenuItem>
-      <Component ref={forwardedRef} {...actionProps}>
+      <Component
+        ref={forwardedRef}
+        {...actionProps}
+        className={classNames(props.className, "transition-none")}>
```

Comment:
transition-colors in button was the cause of border flicker. ( this took a lot of time)